platform/commonUI/edit/test/actions/SaveAsActionSpec.js

```diff
@@ -156,6 +156,28 @@ define(
                 expect(SaveAsAction.appliesTo(actionContext)).toBe(false);
             });
 
+            it("uses the editor capability to save the object", function () {
+                mockEditorCapability.save.andReturn(new Promise(function () {}));
```

Comment:
re: Jasmine 2.3, that dependency is deceptive. The tests are run by karma, and karma-jasmine uses its own version of Jasmine, which is 1.3.1 by the looks of it.

Ok, for now, let's stick with `waitsFor`. The [migration path](http://jasmine.github.io/2.0/upgrading.html#section-Asynchronous_Specs) for these async constructs is well defined, and straightforward.